projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -759,16 +772,176 @@ function wpcom_add_new_page_check( $post_id, $post ) {
 }
 add_action( 'wp_insert_post', 'wpcom_add_new_page_check', 10, 3 );
 
+/**
+ * Return the about page id, if any.
+ *
+ * This fucntion will retrieve the page from the cache whenever possible.
+ *
+ * @return int The page ID of the 'About' page if it exists, null otherwise.
+ */
+function wpcom_get_site_about_page_id() {
+	// First, attempt to get the page ID from the cache.
+	$about_page_id_cache_key = 'wpcom_about_page_id';
+	$about_page_id           = wp_cache_get( $about_page_id_cache_key, 'jetpack_mu_wpcom' );
+
+	if ( false !== $about_page_id ) {
+		$about_page_id = (int) $about_page_id;
+		if ( $about_page_id > 0 ) {
+			return $about_page_id;
+		}
+
+		return null;
+	}
+
+	// Call the current function implementation without any caching logic.
+	$about_page_id = wpcom_find_site_about_page_id(); // could also be wpcom_identify_site_about_page_id() or similar
```

Comment:
Nit: we can remove the trailing comment here.
```suggestion
	$about_page_id = wpcom_find_site_about_page_id();
```